manager/assets/modext/workspace/lexicon/lexicon.grid.js

```diff
@@ -248,30 +247,46 @@ Ext.extend(MODx.grid.Lexicon,MODx.grid.Grid,{
     	this.loadWindow(btn, e, o);
     }
     ,reloadFromBase: function() {
-    	Ext.Ajax.timeout = 0;
-    	var topic = '/workspace/lexicon/reload/';
-        this.console = MODx.load({
-           xtype: 'modx-console'
-           ,register: 'mgr'
-           ,topic: topic
-        });
+        const tb = this.getTopToolbar(),
+            namespace = tb.getComponent('namespace').getValue(),
+            topic = tb.getComponent('topic').getValue(),
+            language = tb.getComponent('language').getValue(),
+            registryTopic = '/workspace/lexicon/reload/';
 
-        this.console.on('complete',function(){
-            this.refresh();
-        },this);
-        this.console.show(Ext.getBody());
+        MODx.msg.confirm({
+            text: _('lexicon_revert_confirm', {
```

Comment:
Consider changing the title for this window, as it's not really a warning. If you elect to take this suggestion, you may even want to create a new `confirm_action` entry that reads “Confirm Action” to make the title even more clear.
```suggestion
            title: _('confirm')
            ,text: _('lexicon_revert_confirm', {
```

Reply:
There is a question in `lexicon_revert_confirm`, so no additional title is required.
Moreover, I did not find a single window in MODX with `title: _('confirm')`, and `title: _('warning')` is used by default, this is enough.